procurement_purchase_requisition_generation/readme/USAGE.md

```diff
@@ -0,0 +1,6 @@
+#. Go to Purchases > Products > Products.
+#. Create a Storable product, set the 'Buy' route and some vendor in the 'Purchase' tab.
+#. Click on the 'Replenish' button.
+#. The 'Buy' route and 'Vendor' field will be defined automatically.
```

Comment:
I don't get this part of the explanation, but I don't see it relevant for this module. The important part that I'm missing is that you have to configure this by product, and you have simply replaced the feature by always creating requisitions.